src/models/responseResolver.js

```diff
@@ -100,6 +100,10 @@ function create (stubs, proxy, callbackURL) {
             }
         });
         return result;
+    }   
+
+    function formatSelector(predicate, matcher, request) {
+        matcher.predicateOperator !== 'exists' ? predicate[matcher.predicateOperator] = request : predicate[matcher.predicateOperator] = matcher.matches;
```

Comment:
Hey Corey - quick review now before I have more time later in the week. I *think* this isn't quite right although the logic is complex enough that I could simply be wrong. The logic for the exists predicate is in line 410 of predicates.js, but it's looking for missing values or empty strings. I think we need that here as well, which means predicate.exists = (typeof request[fieldname] !== 'undefined' && request[fieldname] !== null && request[fieldname] !== ''). If I got the logic right, that would set the predicate value to true if there's a non-empty string in the value, or false if there's no key (e.g. missing query param) or an empty value. 

Similarly, I *think* for the non-exists case, we need request[fieldname], not request.